benchmarks/benchmark_throughput.py

```diff
@@ -229,7 +247,11 @@ def benchmark_throughput(
         )
     elif args.preset == "minimalistic":
         benchmark_throughput(
-            num_experts=1, num_clients=1, num_handlers=1, num_batches_per_client=args.num_batches_per_client
+            num_experts=1,
+            num_clients=1,
+            num_handlers=1,
+            num_batches_per_client=args.num_batches_per_client,
+            batch_size=1024,
```

Comment:
Is changing `batch_size` here intended?

Reply:
This change was needed for checking inputs/outputs streaming in forward/backward. We can change it back to 512 if you want. I don't think there is any effect.

Reply:
Just looked at `master`: there is default `batch_size` for `minimalistic` benchmark. I made it the same here